### PR TITLE
Targeted i18n sync

### PR DESCRIPTION
**Pull request by joshlory**
We:
1. Downloaded the 28 locales we're translating for 6 specific Dance Party files.
2. Ran the sync out step.
3. Manually cleaned up a number of malformed translations.

### CODE REVIEW

**Review by Hamms — APPROVED** [review]

File: dashboard/config/locales/blocks.tr-TR.yml [hunk 1]
```diff
@@ -981,20 +1178,59 @@ tr:
...
+            '"a"': bir
             '"w"': w
             '"s"': s
+            '"d"': d
       gamelab_whenLeftArrow:
         text: sol ok basıldığında
+      gamelab_whenRightArrow:
+        text: sağ ok basıldığında
+      gamelab_whenTouching:
+        text: "{SPRITE1} [SPRITE2]dokunduğunda[1]"
```
Comment (bethanyaconnor) [inline dashboard/config/locales/blocks.tr-TR.yml:1190]: This looks like the restoration got messed up.

File: dashboard/config/locales/blocks.tr-TR.yml [hunk 2]
```diff
@@ -1052,43 +1293,81 @@ tr:
             "{x: 100, y: 300}": sol alt
             "{x: 300, y: 300}": sağ alt
       Mikelab_groupDestroy:
+        text: "{SELECT} {COSTUME} kaldır"
         options:
           SELECT:
             '"all"': tüm
+            '"this"': bu
+      Mikelab_groupDisplace:
+        text: tüm {COSTUME} blok [COSTUME2]hareket eden[1]
```
Comment (bethanyaconnor) [inline dashboard/config/locales/blocks.tr-TR.yml:1302]: Same here

Comment (bethanyaconnor) [reply dashboard/config/locales/blocks.tr-TR.yml:1302]: There are a handful more in this file.

File: dashboard/config/locales/authored_hints.el-GR.yml [hunk 3]
```diff
@@ -898,7 +898,7 @@ el-GR:
           Εάν αυτός ο γρίφος σου φαίνεται απαιτητικός, δοκίμασε να φτιάξεις την συνάρτησή σου λίγο-λίγο κάθε φορά.
           Πρόσθεσε μερικά τουβλάκια και μετά δοκίμασέ το. Πρόσθεσε κι άλλα και δοκίμασέ το πάλι. Γρήγορα, θα τα καταφέρεις!
       Dance_Party_05:
-        validation: "\\*\\* Πώς μπορώ να περάσω αυτό το επίπεδο;\\*\\* \n\n-   Ο χορευτής
+        validation: "**Πώς μπορώ να περάσω αυτό το επίπεδο;** \n\n-   Ο χορευτής
```
Comment (dju90) [inline dashboard/config/locales/authored_hints.el-GR.yml:901]: fixed in crowdin

File: dashboard/config/locales/long_instructions.ar-SA.yml [hunk 4]
```diff
@@ -198,7 +198,7 @@ ar-SA:
 
         يستخدم هذا البرنامج مستوى "جهير" الموسيقى (الأصوات المنخفضة) للتحكم في ارتفاع أحد الراقصين. هل يمكنك جعل الراقص الآخر يستجِب للموسيقى أيضًا؟
       Dance_Party_11: |-
-        يتيح لك المربع البرمجي [0][blockly block] تشغيل الكود البرمجي عند الضغط على السهم لأعلى.
+        يتيح لك المربع البرمجي <xml><block type="Dancelab_whenKey"></block></xml> تشغيل الكود البرمجي عند الضغط على السهم لأعلى.
```
Comment (dju90) [inline dashboard/config/locales/long_instructions.ar-SA.yml]: fixed in Crowdin

File: dashboard/config/locales/long_instructions.id-ID.yml [hunk 5]
```diff
@@ -1327,14 +1327,6 @@ id-ID:
         \-Berapa banyak apel yang gajah kumpulkan?
         \-Berapa banyak apel yang kuda nil kumpulkan?
         \-Berapa banyak waktu yang digunakan untuk mengumpulkan semua apel?
-      Dance_Party_extras_layout_2019: |-
```
Comment (dju90) [inline dashboard/config/locales/long_instructions.id-ID.yml]: fixed in crowdin

File: dashboard/config/locales/long_instructions.ko-KR.yml [hunk 6]
```diff
@@ -916,7 +916,7 @@ ko-KR:
       Dance_Party_extras_layout_2019: |-
         # 형성하자!
 
-        에서 [blockly 블록] 사용이 댄스를 체크 아웃[0] 블록 댄스 걸쳐 백업 댄서 '포메이션을 변경.
+        에서 <xml><block type="Dancelab_layoutSprites" block-text="layout"><title name="GROUP">"MOOSE"</title><title name="FORMAT">???</title></block></xml> 사용이 댄스를 체크 아웃[0] 블록 댄스 걸쳐 백업 댄서 '포메이션을 변경.
```
Comment (dju90) [inline dashboard/config/locales/long_instructions.ko-KR.yml:919]: fixed crowdin

File: dashboard/config/locales/long_instructions.ur-PK.yml [hunk 7]
```diff
@@ -80,7 +80,7 @@ ur-PK:
       Dance_2019_02: |-
         جب اوپر کا ایرو دبایا جاتا ہے تو <xml><block type="Dancelab_whenKey" block-text="when up pressed"></block></xml> بلاک آپ کو کوڈ چلانے دیتا ہے۔ جب آپ دبائیں تو ڈانسر بنانے کے لئے <xml><block type="Dancelab_doMoveEachLR" block-text="do once"><title name="GROUP">"CAT"</title><title name="MOVE">MOVES.XClapSide</title><title name="DIR">-1</title></block></xml> بلاک کا استعمال کریں، یا کوئی اور اقدام آزمائیں!
 
-        _چلائیں بٹن پر کلک کرنے کے بعد اپنے اوپر کے ایرو کو یا [2][] دبانے کو یقینی بنائیں!_
+        _چلائیں بٹن پر کلک کرنے کے بعد اپنے اوپر کے ایرو کو یا ![](<https://images.code.org/419e704e6841c0b696016bf5029b8cb9-image-1571864825506.06.45 PM.png>) دبانے کو یقینی بنائیں!_
```
Comment (dju90) [inline dashboard/config/locales/long_instructions.ur-PK.yml]: fixed crowdin

**Pull request merged**